README.md

```diff
@@ -0,0 +1,32 @@
+# [WIP] Random Coin Toss
+
+> :warning: This repo is still a work in progress - the underlying RandomBeaconHistory is also still a work in progress
```

Comment:
Is the `RandomBeaconHistory` contract in this repo the same core-protocol one? If yes I guess it is not WIP right?

Reply:
You're right, the contract is finalized. I'll update this section